storage-service/src/main/java/com/stnetix/ariaddna/persistence/utils/JPAConfiguration.java

```diff
@@ -29,7 +29,7 @@
     public DataSource dataSource(){
         XmlDbSettingHandler handler = null;
         try {
-            handler = (XmlDbSettingHandler) new XmlParser("settings.xml", new XmlDbSettingHandler()).getHandler();
+            handler = (XmlDbSettingHandler) new XmlParser("./../settings.xml", new XmlDbSettingHandler()).getHandler();
```

Comment:
./../settings.xml this pattern search file in ariadna project working directory.